runtime/src/lib.rs

```diff
@@ -167,7 +167,6 @@ impl<T: Config> OnRuntimeUpgrade for ClearHostConfiguration<T> {
 #[allow(unused_parens)]
 type Migrations = (
 	ClearHostConfiguration<Runtime>,
```

Comment:
To answer the question in this PR's description: no, this migration is not needed anymore and may safely be removed.